### PR TITLE
Allow docblocks to exist on functions.

### LINKED ISSUE #819: Address Literals should be force to be checksummed.
### What's your issue about?
As there is currently not a use case were one would *not* need an address to use upper/lower case checksum. We should enforce this.

Shout out to @pipermerriam ;)
### How can it be fixed?

See the `number` method.

### PR DESCRIPTION
**Pull request by jacqueswww**
### - What I did
Fixes #819 .

### CODE REVIEW

**Comment by fubuloubu** [comment]
Woo!

File: vyper/parser/parser.py [hunk 1]
```diff
@@ -572,7 +572,9 @@ def parse_body(code, context):
         return parse_stmt(code, context)
     o = []
     for stmt in code:
-        o.append(parse_stmt(stmt, context))
+        lll = parse_stmt(stmt, context)
+        if lll:
```
Comment (pipermerriam) [inline vyper/parser/parser.py]: More of a higher level suggestion.  Return value checking is error prone (easy to forget to do).  You might look for an alternate pattern using something like `try/except`.

Comment (jacqueswww) [reply vyper/parser/parser.py]: agreed

**Comment by fubuloubu** [comment]
Woo!

**Pull request merged**